ui/src/api/README.md

```diff
@@ -72,14 +72,18 @@ var DataExplorerService = require('data_explorer_service');
 
```

Comment:
It seems strange to have a bunch of code to execute in the README. Was this autogenerated?

Reply:
Yes. It was in fact very helpful for figuring out how fix above UI problem.